opengrok-web/src/main/webapp/js/utils-0.0.29.js

```diff
@@ -1754,7 +1754,7 @@ function initAutocomplete(config, minisearch) {
 }
 
 function initMinisearchAutocomplete(config) {
-    if (config.allowedFields && !config.allowedFields.includes('full')) {
+    if (config.allowedFields && config.allowedFields.indexOf('full')<0) {
```

Comment:
there should be a space `ull')<0)` -> `ull') < 0)`

Reply:
thx, fixed